source/includes/_evidence.md

```diff
@@ -18,8 +18,8 @@ Mind that PO1 has a main question which can be answered as "Yes" or "No". We are
 The number at the end of the endpoint indicates the evidence number. This number is increased incrementally as you add multiple evidence hyperlinks to an indicator {variable}. In the following examples the first evidence item will be hyperlink, and second one will be file. Even if we remove the hyperlink evidence the endpoint and the item number would remain the same for the evidence file as reference.
 </aside>
 
-## POST values/{variable} - Indicate File Type
-Indicate filetype as hyperlink by sending `H` as the value.
+## POST values/{variable} - Indicate Evidence Type for Hyperlink
```

Comment:
Just to be sure - for all section title changes make sure to look through the `source/includes` folder if it isn't referred somewhere. References will break when title is changed.

Not exactly sure anymore how sections are named but an example of a reference is the one you made in https://github.com/GRESB/api-docs/pull/147/files#diff-c2ff44acd13b091fc6fbb590050377315664eba3a55148d252da77ecfbf3baebR27.

Reply:
See you got the ones in the endpoint table 👍 https://github.com/GRESB/api-docs/pull/147/files#diff-488fcfd19ec590bb77baacb451ccecab539e2f14658bce0fab660b948e77a6e8R34